scripts/setup_integration_env.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -o pipefail
+
+# setup_integration_env.sh
+#
+# SUMMARY
+#
+#  Sets up Vector integration test environments
+
+set -x
+
+if [ $# -ne 2 ]
+then
+    echo "Usage: $0 {integration_test_suite} {stop|start}" 1>&2; exit 1;
+    exit 1
+fi
+INTEGRATION=$1
+ACTION=$2
+
```

Comment:
Since this might be run stand-alone from the command line, it would probably be a good idea to check that `$CONTAINER_TOOL` is actually set here, and default it to our preferred tool.